### ISSUE TITLE
There is a CVE on the version of ansible we are pinned to

### ISSUE THREAD

**Issue by heartsucker** [freedomofpress/securedrop#2450]
# Bug

## Description

```json
[
  "ansible", 
  "<2.3.1", 
  "2.2.3.0", 
  "ansible before 2.3.1 is vulnerable to CVE-2017-7481 - data for lookup plugins used as variables was not being correctly marked as \"unsafe\".", 
  "34941"
], 
```

https://access.redhat.com/security/cve/CVE-2017-7481

## Steps to Reproduce

```
pip install safety
safety check --json -r securedrop/requirements/develop-requirements.txt
```

## Expected Behavior

We shouldn't have CVEs in our code.

## Actual Behavior

We do. :(

**Comment by conorsch** [comment]
Thanks, @heartsucker. Historically we were not good about updating the Ansible version regularly (#1146), but now that we're using virtualenv the story is much better. Let's bump to 2.3.1 since it's safe—from minimal testing, 2.4 will require changes, and is best handled separately.

**Comment by kushaldas** [comment]
Tried a fresh install with 2.3.2. Installation worked well. Manually testing the application now.

**Comment by kushaldas** [comment]
Things seem to be working fine as I can see in the manual testing.

**Comment by heartsucker** [comment]
One note: some of our dependencies in `testinfra/requirements.txt` force the Ansible version to `< 2.3`. We need to find the culprit.

**Comment by dachary** [comment]
IIRC the problem is with install_files/ansible-base/action_plugins/synchronize.py which break with 2.3

**Comment by redshiftzero** [comment]
Fixed in SecureDrop 0.4.4 in commit b58f553
